pkgs/development/python-modules/pyqt/pyqt5-fix-dbus-mainloop-support.patch

```diff
@@ -1,27 +1,5 @@
-From 944d5467e1655aac20a14325631df6daccaf5804 Mon Sep 17 00:00:00 2001
-From: Jan Tojnar <jtojnar@gmail.com>
-Date: Sun, 3 Mar 2019 01:13:46 +0100
-Subject: [PATCH] Fix building on Nix
-
-./configure.py tries to find dbus-python header in dbus-1 includedir
-obtained from pkg-config or from --dbus flag. Unfortunately, when supplied,
-it also uses the flag for locating dbus-1 headers. This fails on Nix,
-since every package is installed into its own immutable tree so we cannot
-use a single directory for both dbus-python and dbus-1. We can fix this by
-using pkg-config for finding dbus-python headers too.
-
-Additionally, the build system also tries to install the dbus support module
-to dbus-python tree. Often, it is possible to handle this in pkgconfig as well [1]
-but unfortunately, dbus-python does not export the moduledir in its pc file
-so I have decided to solve this with an extra configure flag.
-
-[1]: https://www.bassi.io/articles/2018/03/15/pkg-config-and-paths/
```

Comment:
It would be nice to preserve this explanation.

Reply:
Done